src/reducers/entities/posts.js

```diff
@@ -103,6 +132,16 @@ function handlePendingPosts(pendingPostIds = [], action) {
         }
         return nextPendingPostIds;
     }
+    case PostTypes.REMOVE_PENDING_POST: {
+        const postId = action.data.id;
+        const nextPendingPostIds = [];
+        for (const post of pendingPostIds) {
+            if (post !== postId) {
+                nextPendingPostIds.push(post);
+            }
+        }
+        return nextPendingPostIds;
```

Comment:
Same here, you don’t need to loop, just remove the post id from the array

Reply:
Same here.